web/packages/core/src/ruffle-player.ts

```diff
@@ -1141,23 +1144,36 @@ export class RufflePlayer extends HTMLElement {
 
         const errorText = errorArray.join("");
 
-        // Remove query params for the issue title.
-        const pageUrl = document.location.href.split(/[?#]/)[0];
-        const issueTitle = `Error on ${pageUrl}`;
-        let issueLink = `https://github.com/ruffle-rs/ruffle/issues/new?title=${encodeURIComponent(
-            issueTitle
-        )}&template=error_report.md&labels=error-report&body=`;
-        let issueBody = encodeURIComponent(errorText);
-        if (
-            errorArray.stackIndex > -1 &&
-            String(issueLink + issueBody).length > 8195
-        ) {
-            // Strip the stack error from the array when the produced URL is way too long.
-            // This should prevent "414 Request-URI Too Large" errors on Github.
-            errorArray[errorArray.stackIndex] = null;
-            issueBody = encodeURIComponent(errorArray.join(""));
+        const buildDate = new Date("%BUILD_DATE%");
+        const monthsPrior = new Date();
+        monthsPrior.setMonth(monthsPrior.getMonth() - 6); // 6 months prior
+        const isBuildOutdated = monthsPrior > buildDate;
+
+        // Create a link to GitHub with all of the error data, if the build is not outdated.
+        // Otherwise, create a link to the downloads section on the Ruffle website.
+        let issueLink;
+        if (!isBuildOutdated) {
+            // Remove query params for the issue title.
+            const pageUrl = document.location.href.split(/[?#]/)[0];
+            const issueTitle = `Error on ${pageUrl}`;
+            issueLink = `https://github.com/ruffle-rs/ruffle/issues/new?title=${encodeURIComponent(
+                issueTitle
+            )}&template=error_report.md&labels=error-report&body=`;
+            let issueBody = encodeURIComponent(errorText);
+            if (
+                errorArray.stackIndex > -1 &&
+                String(issueLink + issueBody).length > 8195
+            ) {
+                // Strip the stack error from the array when the produced URL is way too long.
+                // This should prevent "414 Request-URI Too Large" errors on GitHub.
+                errorArray[errorArray.stackIndex] = null;
+                issueBody = encodeURIComponent(errorArray.join(""));
+            }
+            issueLink += issueBody;
+            issueLink = `<a target="_top" href="${issueLink}">Report Bug</a>`;
```

Comment:
IMO it's quite confusing to use `issueLink` first as a URL, then as an HTML tag. How about introducing a new `linkTitle` variable that will hold either `"Report Bug"` or `"Update Ruffle"`, then using it down below as following
```js
<li><a target="_top" href="${issueLink}">${linkTitle}</a></li>
```
?